### ISSUE TITLE
[Data Objects] Links in Blocks will not be cleaned up when Target Document is deleted

### ISSUE THREAD

**Issue by putzflorian** [pimcore/pimcore#7369]
If the target document is deleted from a link nested in a block, you can no longer save the object

### Error:
```
Validation failed:  (invalid internal link, referenced document with id [201] does not exist
```

![linksinblock](https://user-images.githubusercontent.com/1086343/96677234-79299d00-136f-11eb-91dc-3914a113eb78.png)

**Comment by brusch** [comment]
Resolved by #7381

**Comment by putzflorian** [comment]
thx
